### PR TITLE
Update video uploader preview poster image selection notice

### LINKED ISSUE #31988: Jetpack VideoPress upload - Clarification of Poster Image selection notice
### Impacted plugin

VideoPress

### What

The notice beneath the video when it is first added, for choosing a poster image is not very clear. It should say ‘Use Slider in Block settings (Poster) to choose a thumbnail’.

### How

Replacing the text, "Use the slider to choose a poster image" beneath the uploading video, with something like, "Use Slider in Block settings (Poster) to choose a thumbnail" instead.

Poster image selection notice when uploading video to VideoPress

Alternatively, it can be removed altogether, or the user instructed to view the Block settings for any changes they need to make.

### PR DESCRIPTION
**Pull request by danmaze**
Fixes #31988

## Proposed changes:

The notice previously read "Use the slider to choose a poster image", but there are cases where the video preview doesn't show up, perhaps due to some kind of latency making the notice somewhat confusing, as shown in the screenshot in the issue comment.

This PR updates the notice to further clarify that you can also change the poster image from the Block Settings sidebar under the Poster section:

videopress-poster-image-selection-notice-update

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Does this pull request change what data or activity we track or use?

No, it does not.

## Testing instructions:

1. Build the VideoPress package and activate the plugin.
2. Add the VideoPress block to a post or page.
3. Upload a video.
4. Notice the updated upload/processing preview and poster image selection notice.

### CODE REVIEW

**Comment by danmaze** [comment]
I suspect that the help message might be too long now. Maybe something like "Use the slider to choose a poster image or change it from the block settings" instead?

File: projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/uploader-editor.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ const PosterActions = ( { hasPoster, onSelectPoster, onRemovePoster } ) => {
 	return (
 		<span className="uploading-editor__scrubber-help">
 			{ __(
-				'This is how the video will look. Use the slider to choose a poster image.',
+				'This is how the video will look. Use the slider to choose a poster image or adjust it in the Block Settings Sidebar under Poster.',
```
Comment (jeherve) [inline projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/uploader-editor.js]: afaik the section is named "poster and preview". Should we adjust wording based on that?

Comment (danmaze) [reply projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/uploader-editor.js]: checked and I see it's called "Poster and preview" in Calypso but simply "Poster" in WP Admin. I vote to change the direction to "Use the slider to choose a poster image or change it from the block settings" which is agnostic while still communicating that the poster image can be changed from the block settings

Comment (jeherve) [reply projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/uploader-editor.js]: Yeah, I like that option

**Review by jeherve — APPROVED** [review]

**Pull request merged**